cloudify_cli/table.py

```diff
@@ -136,13 +143,13 @@ def print_details(data, title):
     """
     if get_global_json_output():
         output(json.dumps(data, cls=CloudifyJSONEncoder))
-    else:
-        output(title)
-        for item in data.items():
-            field_name = str(item[0]) + ':'
-            field_value = str(item[1])
-            field_value = get_timestamp(field_value) or field_value
-            output('\t{0} {1}'.format(field_name.ljust(16), field_value))
+        return
```

Comment:
Any reason we prefer changing this to use a return instead of an else? I think the else reads more nicely, and this appears to otherwise be a bunch of changes that are unrelated to the actual change?